src/init.cpp

```diff
@@ -625,8 +625,15 @@ void ThreadStartWalletNotifier()
         if (walletBestBlockHash.has_value()) {
             int64_t slept;
             auto timedOut = [&]() -> bool {
-                MilliSleep(50);
-                slept += 50;
+                MilliSleep(500);
+                {
+                    LOCK(cs_main);
+                    bool isReindexing{false};
+                    if (pblocktree->ReadReindexing(isReindexing) && !isReindexing) {
```

Comment:
(not blocking) This causes a db read; perhaps once `isReindexing` is `false` once we can stop calling `ReadReindexing`.